### DISCUSSION TITLE
Installing to CentOS 8

### DISCUSSION THREAD

**Discussion by SydneyMae** [Pryaxis/TShock#2175] [Q&A]
How do I install TShock for the latest version (v1.4.1.2)? All of tutorials I found are for CentOS 6. Any knowledgebase articles or anything like that would be greatly appreciated!

**Comment by Kojirremer** [comment]
I haven't tried that OS before, but what I can tell you is that you have to be extra sure to install the latest mono version from their website, since hosted repo versions tend to be outdated and TShock needs the recent builds.
